docs/book/language-reference.md

```diff
@@ -47,6 +47,8 @@ complex types.
 | <span class="opa-keep-it-together">``s3 = s1 & s2``</span> | 2 | ``s3`` is the intersection of ``s1`` and ``s2``. |
 | <span class="opa-keep-it-together"><code>s3 = s1 &#124; s2</code></span> | 2 | ``s3`` is the union of ``s1`` and ``s2``. |
 | <span class="opa-keep-it-together">``s3 = s1 - s2``</span> | 2 | ``s3`` is the difference between ``s1`` and ``s2``, i.e., the elements in ``s1`` that are not in ``s2`` |
+| <span class="opa-keep-it-together">``intersection(set_collection, output)``</span> | 1 | ``output`` is the intersection of the sets in the ``set collection``  |
```

Comment:
Instead of set collection we could use some notation like `set[set]`?

Reply:
Replaced `set collection` by `set[set]`